src/swaps/thorchain/ThorchainSwapProvider.ts

```diff
@@ -267,14 +259,36 @@ class ThorchainSwapProvider extends SwapProvider {
       SAFE_FEE_MULTIPLIER
     );
     const toAmountInUnit = currencyToUnit(cryptoassets[to], baseToAsset(swapOutput).amount());
+
+    // Substract swap.receiveFee from toAmount as this is the minimum limit that you are going to receive
+    const baseOutputAmount = baseAmount(toAmountInUnit.minus(receiveFeeInUnit), cryptoassets[to].decimals);
+    const slippageCoefficient = new BN(1).minus(slippage);
+    const minimumOutput = baseOutputAmount.amount().multipliedBy(slippageCoefficient).dp(0);
+
+    return minimumOutput;
+  }
+
+  async getQuote(quoteRequest: QuoteRequest) {
+    const { from, to, amount } = quoteRequest;
+    // Only ethereum, bitcoin and bc chains are supported
+    if (!SUPPORTED_CHAINS.includes(cryptoassets[from].chain) || !SUPPORTED_CHAINS.includes(cryptoassets[to].chain))
+      return null;
+
+    // Slippage must be higher for small value swaps
+    const min = await this.getMin(quoteRequest);
+    const slippage = new BN(amount).gt(min.times(2)) ? 0.03 : 0.05;
```

Comment:
why min * 2?
why 3 & 5%?

Reply:
`min * 2` is just a qualifier for a small vs bigger swap size
The default slippage is 3% (I found this suggestion from asgardex), 5% is a slippage to give smaller value swaps a higher chance of succeeding
I feel the comment above it describes this in short